src/components/ShellView/ResultTable.vue

```diff
@@ -40,20 +40,55 @@
             </th>
           </tr>
         </thead>
-        <tbody>
-          <tr v-for="(row, i) in rows" :key="i">
-            <td v-for="(cell, j) in row" :key="j">
-              <ul v-if="Array.isArray(cell)" class="list-group">
-                <li v-for="(item, k) in cell" :key="k" class="list-group-item">
-                  <b>{{ item.name }}:</b> {{ item.value }}
+      <tbody>
+      <tr v-for="(row, i) in rows" :key="i">
+        <td v-for="(cell, j) in row" :key="j">
+          <div v-if="Array.isArray(cell)" class="list-container">
+            <ul class="list-group">
+              <li v-for="(item, k) in cell" :key="k" class="list-group-item">
+                <b>{{ item.name }}:</b> {{ item.value }}
+              </li>
+            </ul>
+          </div>
+          <div v-else-if="cell.hasOwnProperty('_nodes') && cell.hasOwnProperty('_rels')" class="data-container">
+            <div class="flex-item">
+              <ul class="list-group">
+                <li v-for="(item, k) in cell._nodes" :key="k" class="list-group-item-o">
+                  <ul class="list-group">
+                    <li v-for="(field, k) in item" :key="k" class="list-group-item">
+                      <span v-if="field.name === 'LABEL'">
```

Comment:
Since you always place the field name at the first, you should not use `v-if` in this case. It should be something like:
```
<li>
  <span> {{ LABEL }} </span>
<li>

<li v-for>
  <span> {{ other fields }} </span>
</li>
```

Also, you should keep using `_label` as key as it is reserved.